halo2_proofs/src/poly/commitment.rs

```diff
@@ -160,6 +160,11 @@ impl<C: CurveAffine> Params<C> {
         self.g.clone()
     }
 
+    /// The `g_lagrange` bases
+    pub fn g_lagrange(&self) -> Vec<C> {
```

Comment:
Document as an addition in the changelog. It would also be useful to document how or why this is useful. If it is only useful for endoscaling:
```suggestion
    #[cfg(feature = "unstable-endoscale-gadget")]
    pub fn g_lagrange(&self) -> Vec<C> {
```